examples/globe_wfs_extruded.js

```diff
@@ -35,11 +36,17 @@ promises.push(itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').th
 function altitudeLine(properties, contour) {
     var altitudes = [];
     var i = 0;
-    var alt = 0;
+    var result;
+    var tile;
+    var layer = globeView.wgs84TileLayer;
     if (contour.length && contour.length > 0) {
         for (; i < contour.length; i++) {
-            alt = itowns.DEMUtils.getElevationValueAt(globeView.wgs84TileLayer, contour[i]).z + 2;
-            altitudes.push(alt);
+            result = itowns.DEMUtils.getElevationValueAt(layer, contour[i], 0, tile);
```

Comment:
It saddens me that you need to reiterate later if no result have been found with cache. Even if it is only an example, `getElevationValueAt` should be handling this on its own.

Reply:
yes, I'll do another PR